src/main/java/teammates/ui/controller/Action.java

```diff
@@ -576,7 +577,23 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
                && loggedInUser.googleId != null
                && !loggedInUser.googleId.equals(requestedUserId);
     }
-    
+
+    protected static List<FeedbackParticipantType> getParticipantListFromParams(String participantListParam) {
```

Comment:
I believe @wkurniawan07 is referring to your abstraction of this method into the Action class. Perhaps it should be placed in another abstract class like in #6480